ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+/**
+ * @module Page::MessagesList
+ * Page::MessagesList components are used to display breadcrumb links. This is component will be replaced when HDS system is incorporated
+ *
+ * @example
+ * ```js
+ * <Page::MessagesList @messages={{this.messages}}  />
+ * ```
+ * @param {array} messages - array message objects
+ */
+
+export default class MessagesList extends Component {
+  @service store;
+
+  get mountPoint() {
```

Comment:
Claire and I went back and forth on mountPoint for kv. It's still in there, but there's also a ticket to  take it out. It's really up to you, but if you think it's easier to just hard code the value instead of calling this getter, go with that flow. Again, whatever is going to be easier for the next person to read.

Reply:
How would you invoke getOwner directly in the template to get the mountPoint?